projects/1.16.1/assets/industrial-foregoing/industrialforegoing/lang/zh_cn.json

```
@@ -1,455 +1,531 @@
+  "text.industrialforegoing.pull": "吸入",
+  "text.industrialforegoing.maxy": "最大 Y:",
+  "text.industrialforegoing.miny": "最小 Y:",
+  "text.industrialforegoing.weight": "重量:",
+  "text.industrialforegoing.biomes": "生物群系",
+  "text.industrialforegoing.leaf_shearing_addon": "* 可以使用叶片剪切插件",
+  "text.industrialforegoing.can_harvest": "* 可以收获:",
+  "text.industrialforegoing.plant.any": "- 任何Minecraft原版风格的作物",
+  "text.industrialforegoing.plant.nether_wart": "- 地狱疣",
+  "text.industrialforegoing.plant.chorus": "- 紫颂植物",
+  "text.industrialforegoing.plant.sugar_cane": "- 甘蔗§o(不需要补种)",
+  "text.industrialforegoing.plant.cactus": "- 仙人掌§o(不需要补种)",
+  "text.industrialforegoing.plant.melon": "- 西瓜§o(不需要补种)",
+  "text.industrialforegoing.plant.pumpkin": "- 南瓜§o(不需要补种)",
+  "text.industrialforegoing.plant.any_tree": "- 任何Minecraft原版风格的树",
+  "text.industrialforegoing.plant.slime_tree": "- Tinker's construct(匠魂)的史莱姆树",
+  "__comment": "!-- PAGE FORMAT HELP",
+  "__comment1": "@L@\" -> Adds a new line",
+  "__comment2": "@PAGE@\" ->将创建一个新页面，以下文本将位于新页面中",
+  "__comment3": "{\" ->将启动一个粗体/彩色字符串以突出显示",
+  "__comment4": "}\" -> 将停止高亮显示 -->",
+  "text.industrialforegoing.book.title": "工业先锋手册",
+  "text.industrialforegoing.book.landing": "如果你需要让一切都自动化！",
+  "text.industrialforegoing.book.category.getting_started": "入门",
+  "text.industrialforegoing.book.category.generators": "发电设备",
+  "text.industrialforegoing.book.category.agriculture": "农业设备",
+  "text.industrialforegoing.book.category.animal_husbandry": "畜牧业设备",
+  "text.industrialforegoing.book.category.magic": "魔法设备",
+  "text.industrialforegoing.book.category.storage": "存储设备",
+  "text.industrialforegoing.book.category.mob_interaction": "怪物交互设备",
+  "text.industrialforegoing.book.category.resource_production": "资源生产设备",
+  "text.industrialforegoing.book.category.items": "物品设备",
+  "text.industrialforegoing.book.category.conveyors": "传送带",
+  "text.industrialforegoing.book.sewer": "提供能量后，它会从其顶部附近的动物中收集{污水}。",
+  "text.industrialforegoing.book.animal_growth_increaser": "提供能量和{养殖所需食物}后，它就会喂养小的动物使它们更快的{生长}。",
+  "text.industrialforegoing.book.animal_independence_selector": "提供能量后，它会将{未成年}动物移到机器工作区域的后方。@L@@L@如果安装{成年生物过滤插件}那么它会将成年的动物移到机器工作区域的后方，而不是未成年动物。",
+  "text.industrialforegoing.book.animal_resource_harvester": "提供能量后，这个机器将尝试挤牛奶或剪羊毛，并且你不需要提供{桶}和{剪刀}。@L@@L@它也能从鱿鱼中获得{墨囊}。",
+  "text.industrialforegoing.book.animal_stock_increaser": "当提供{繁殖所需食物}和能量后，如果工作区域内的动物可以被繁殖，那么这个机器会尽量使区域内的动物繁殖。@L@@L@工作区域内同一时间最多只能有{35}只动物。",
+  "text.industrialforegoing.book.biofuel_generator": "使用生物燃料产生能量。{一桶}生物燃料将产生160RF/t，总共{640,000}RF。总共{28}台发电机足够使用1台{生物质炉}进来供应生物燃料。",
+  "text.industrialforegoing.book.bioreactor": "提供{能量}和{生物质材料}将产生{生物燃油}.@L@@L@每种不同的物品将提高{10}mb的提取效率，当达到最大效率时，炉内每次反应将会产生{1440}mb生物燃油并消耗{9}种不同的生物质材料。",
+  "text.industrialforegoing.book.black_hole_controller": "这个机器内可以放置9个{黑洞存储器}并访问内部存储的物品。这意味着可以存储{2,147,483,647}个物品的9倍。",
+  "text.industrialforegoing.book.black_hole_tank": "它可以存储{2,147,483,647}mb的同种流体。",
```

```suggestion
  "text.industrialforegoing.book.black_hole_tank": "它可以存储 {2,147,483,647}mb 同种流体。",
```
参见：https://rules.cfpa.team/
下面同理